java/c/src/test/java/org/apache/arrow/c/ArrowArrayUtilityTest.java

```diff
@@ -50,6 +50,14 @@ void afterEach() {
     allocator.close();
   }
 
+  @Test
+  void arraySchemaInit {
+    ArrowSchema schema = ArrowSchema.allocateNew(allocator);
+    Snapshot snapshot = schema.snapshot();
+    assertThat(snapshot.release).isEqualTo(0);
+    schema.close();
+  }
```

Comment:
Not sure why there is style error in CI. The diff looks good, but in CI it is `voidarraySchemaInit`

```
Error:  Failed to execute goal org.apache.maven.plugins:maven-checkstyle-plugin:3.4.0:check (validate) on project arrow-c-data: Failed during checkstyle configuration: Exception was thrown while processing /build/java/c/src/test/java/org/apache/arrow/c/ArrowArrayUtilityTest.java: IllegalStateException occurred while parsing file /build/java/c/src/test/java/org/apache/arrow/c/ArrowArrayUtilityTest.java. 54:23: no viable alternative at input 'voidarraySchemaInit{': NoViableAltException -> [Help 1]
```

Reply:
It looks wrong here. The parentheses are missing.